CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+CONTRIBUTING# Contributing
```

Comment:
This is improperly formatted.

Please just base this file on the repo: https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/CONTRIBUTING.md